src/Service/Modules/LoanHeadersModule.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Linn.Stores.Service.Modules
+{
+    using Linn.Stores.Facade.Services;
+    using Linn.Stores.Resources.RequestResources;
+    using Linn.Stores.Service.Models;
+
+    using Nancy;
+    using Nancy.ModelBinding;
+
+    public sealed class LoanHeadersModule : NancyModule
```

Comment:
This should just be called LoanModule, LoanService etc. There shouldn't really be a loan detail repository. 
I know you didn't make that but ideally we should really refactor it out and replace with this one.